builtin/registry/Makefile

```diff
@@ -55,7 +55,9 @@ depend:
 clean:
 	$(call CLEAN)
 
-distclean: clean
+clean_context:
 	$(call DELFILE, *.bdat)
 	$(call DELFILE, *.pdat)
+
+distclean: clean
```

Comment:
need depend on clean_context too?

Reply:
i don't think it's necessary here as the top level Makefile have it